### PR TITLE
[ux] Night Mapping theme improvements follow-up

### PR DESCRIPTION
**Pull request by gacarrillor**
+ [x] Make sure text on ProgressBars is visible also for Night Mapping theme.
 + [x] Model (.ili) icon

Some screenshots:

Screenshot from 2024-06-24 10-25-51

image

image

image

Before (Model icon):

image

--------------

Follow-up #928

### CODE REVIEW

**Comment by gacarrillor** [comment]
Regarding the `LayerModel` items, this is how they look like in non-default QGIS themes:

image

image

The `LayerModel.data()`'s returned value for the `Qt.BackgroundRole` role is being overridden by a global stylesheet set by the non-default QGIS themes (like `Night Mapping` theme):

```
QTreeView::item, QTreeView::branch {
	background: transparent;
	color: @textlight;
}
```

One might expect that even having a global stylesheet, one could set the item background via Qt.BackgroundRole, but the current behavior is well known in Qt and considered as a feature, rather than a bug. For reference, some related links:

https://forum.qt.io/topic/95940/model-backgroundrole-overridden-by-style-sheet
https://bugreports.qt.io/browse/QTBUG-70100

## Suggestion
Instead of using colors, add a text clue for non-INTERLIS layers (e.g., `My Layer*`) and in the legend below the `QTreeView`, replace color symbols by something like `*: Non-INTERLIS layer`.

**Comment by gacarrillor** [comment]
What's missing couldn't be solved due to a Qt bug and was posted as an issue: https://github.com/opengisch/QgisModelBaker/issues/939

File: QgisModelBaker/images/file_types/model.png
Comment (signedav) [inline QgisModelBaker/images/file_types/model.png:1]: Which one is the used one?

File: QgisModelBaker/utils/gui_utils.py [hunk 1]
```diff
@@ -115,45 +115,57 @@ class DropMode(Enum):
...
 
 # style
-ORANGE = "#D1C28E"
+ORANGE = "#D1C28E"  # Not used
 GREEN = "#A1DE9B"
 BLUE = "#9BCADE"
-PURPLE = "#B18BC9"
-RED = "#EBB3A4"
+PURPLE = "#B18BC9"  # Not used
+RED = "#ff0000" if QgsApplication.themeName() == "Night Mapping" else "#EBB3A4"
```
Comment (signedav) [inline QgisModelBaker/utils/gui_utils.py:122]: Can we have softer colors here. Similar to the bright theme. While there we had more white in it (pastel colors) maybe with the dark more grayish colors could work...

File: QgisModelBaker/images/file_types/model.png
Comment (gacarrillor) [reply QgisModelBaker/images/file_types/model.png:1]: Sorry, I uploaded the wrong icon version.
Just updated it and added a before screenshot for comparison.

File: QgisModelBaker/utils/gui_utils.py (hunk 1 above)
Comment (gacarrillor) [reply QgisModelBaker/utils/gui_utils.py:122]: The orange color is also used by the palette, and actually used in all toolbars for intermediate steps.

Would this red work for you?

image

What about this green color? (Compare it with the selected dataset color)

image

Comment (signedav) [reply QgisModelBaker/utils/gui_utils.py:122]: Yeah this is nice. The other made me leave through the emergency escape everytime when the validation failed ;-)

Orange and green is fine. Thanks.

**Review by signedav — APPROVED** [review]

**Comment by signedav** [comment]
Here a nice screeny maybe for the documenation:
Screenshot from 2024-06-27 12-18-24

Looks really nice!

**Comment by gacarrillor** [comment]
> Here a nice screeny maybe for the documenation

Nice! Yeah, it would be cool to advertise multi-theme support.

**Pull request merged**